python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-adaptive-virtual-topology.j2

```diff
@@ -8,7 +8,11 @@
 !
 router adaptive-virtual-topology
 {%     if router_adaptive_virtual_topology.topology_role is arista.avd.defined %}
-   topology role {{ router_adaptive_virtual_topology.topology_role }}
+{%         set topology_role = "topology role " ~ router_adaptive_virtual_topology.topology_role %}
+{%         if router_adaptive_virtual_topology.gateway_vxlan is arista.avd.defined(true) %}
+{%             set topology_role = topology_role ~ " gateway vxlan" %}
+{%         endif %}
```

Comment:
is `gateway vxlan` applicable (and valid EOS config) for pathfinders?

Reply:
good catch